### ISSUE TITLE
Feature request: Preview base 64 strings in the resonse

### ISSUE THREAD

**Issue by joankaradimov** [graphql/graphiql#992]
The query window shows you a tooltip with helpful information when you hover over certain query tokens. Since using base64 encoded strings for IDs is common, it might be nice to do something similar for them -- a tooltip with the decoded contents of any base64-looking string in the response window.

**Comment by benjie** [comment]
This sounds like a great candidate for a plugin in the new plugin system we'll be building. Thanks for submitting your issue.

**Comment by acao** [comment]
yes  this is brilliant. makes sense alongside the existing image preview
